frontend/src/app/animals-list/animals-list.component.html

```diff
@@ -1,5 +1,6 @@
 <div class="content" *ngIf="animals$ | async; let animals">
   <app-animal-filter (filtering)="filterByName($event)" [countResult]="animals.length"></app-animal-filter>
```

Comment:
Maybe it would be better to rename `app-animal-filter` to something like `app-animal-name-filter`. Because `app-animal-filter` now doesn't look informative